src/sinol_make/helpers/compiler.py

```diff
@@ -25,7 +25,7 @@ def get_c_compiler_path():
 		else:
 			return 'gcc'
 	elif sys.platform == 'darwin':
-		for i in range(9, 13):
+		for i in range(9, 11):
```

Comment:
`for i in [10, 9]:` is both more readable and first tries the newer compiler version (because the version 10 is the preferred one, do I remember correctly?).

Reply:
Version 9 is currently used in oioioi, but it isn't available on MacOS with ARM.

Reply:
Ok, then let's leave it with the order `9, 10`.